games-puzzle/tworld/tworld-1.3.2.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Emulation of the game Chip's Challenge"
+DESCRIPTION="Tile World is an emulation of the game Chip's Challenge. \
+Chip's Challenge was originally written for the Atari Lynx by Chuck Sommerville, \
+and was later ported to MS Windows by Microsoft (among other ports). \
+Chip's Challenge is a game made up of both intellectually engaging puzzles \
+and situations demanding fast reflexes. As you might have guessed, \
+it is a tile-based game. The object of each level is simply to get out — i.e., \
+to find and achieve the exit tile. This simple task, however, \
+can sometimes be extremely challenging."
+HOMEPAGE="http://www.muppetlabs.com/~breadbox/software/tworld/"
+COPYRIGHT="2001-2015 Brian Raiter"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI=http://www.muppetlabs.com/~breadbox/pub/software/tworld/tworld-$portVersion-CCLPs.tar.gz"
```

Comment:
Why not use the default archive? http://www.muppetlabs.com/~breadbox/pub/software/tworld/tworld-1.3.2.tar.gz

Reply:
PS, missing a " before http

Reply:
@Begasus That one doesn't come with any levels and since Haiku has read-only filesystem, I think that the archive I have chosen with level preinstalled will be nice :)